Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
+    domain_name (0.6.20240107)
+    doorkeeper (5.6.8)
       railties (>= 5)
-    dotenv (2.2.1)
+    dotenv (2.8.1)
     encryptor (3.0.0)
     erubi (1.12.0)
-    erubis (2.7.0)
-    execjs (2.7.0)
-    faraday (2.7.11)
+    execjs (2.9.1)
+    faraday (2.8.1)
       base64
       faraday-net_http (>= 2.0, < 3.1)
       ruby2_keywords (>= 0.0.4)
-    faraday-http-cache (2.5.0)
+    faraday-http-cache (2.5.1)
       faraday (>= 0.8)
     faraday-multipart (1.0.4)
       multipart-post (~> 2)
     faraday-net_http (3.0.2)
     faraday-net_http_persistent (2.1.0)
       faraday (~> 2.5)
       net-http-persistent (~> 4.0)
-    ffi (1.15.5)
+    ffi (1.16.3)
     ffi-compiler (1.0.1)
       ffi (>= 1.0.0)
       rake
-    flay (2.12.0)
-      erubis (~> 2.7.0)
+    flay (2.13.2)
+      erubi (~> 1.10)
       path_expander (~> 1.0)
       ruby_parser (~> 3.0)
       sexp_processor (~> 4.0)
     flowdock (0.7.1)
       httparty (~> 0.7)
       multi_json
-    forking_test_runner (1.2.0)
+    forking_test_runner (1.13.0)
       parallel_tests (>= 1.3.7)
-    git_diff_parser (3.1.0)
-    gitlab (4.10.0)
-      httparty (~> 0.14, >= 0.14.0)
-      terminal-table (~> 1.5, >= 1.5.1)
-    globalid (1.1.0)
-      activesupport (>= 5.0)
+    git_diff_parser (3.2.0)
+    gitlab (4.19.0)
+      httparty (~> 0.20)
+      terminal-table (>= 1.5.1)
+    globalid (1.2.1)
+      activesupport (>= 6.1)
     goldiloader (3.2.0)
       activerecord (>= 4.2, < 6.3)
       activesupport (>= 4.2, < 6.3)
-    hashdiff (0.3.4)
+    hashdiff (0.4.0)
     hashie (5.0.0)
-    http (5.1.1)
+    http (5.2.0)
```

Comment:
### [http](https://github.com/httprb/http)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 5.1.1 → 5.2.0

[_[change-log](https://github.com/httprb/http/blob/v5.2.0/CHANGELOG.md), [source-code](https://github.com/httprb/http), [gem-diff](https://my.diffend.io/gems/http/5.1.1/5.2.0)_]





<details>
<summary>Commits</summary>

A change of **18** commits. See the full changes on [the compare page](https://github.com/httprb/http/compare/v5.1.1...v5.2.0).

These are the first 10 commits:
- (32a2ae6) [Update CHANGES.md (#738)](https://github.com/httprb/http/commit/32a2ae68a1650d267f0687381d58880cf1b60315)
- (97aa636) [Add `Connection#finished_request?` (#743)](https://github.com/httprb/http/commit/97aa636043c6eba348556c1b04b79f228d14be78)
- (ed8bd1f) [Bump llhttp-ffi (0.5.0) (#744)](https://github.com/httprb/http/commit/ed8bd1f6ea49d5add8baeec25c6a079b1749540a)
- (73a59f5) [Let the instrumenter log exceptions too (#746)](https://github.com/httprb/http/commit/73a59f5af47db6c9b943f1dbf628c2a84b7e4662)
- (4e0696c) [Run CI against Ruby 3.2 (#742)](https://github.com/httprb/http/commit/4e0696c6596123c122389402234de057fcc68550)
- (462d711) [Update README.md (#751)](https://github.com/httprb/http/commit/462d711040735be3661068e9e52eb14a4269f1f4)
- (9bfcdfb) [Downcase Content-Type charset name (#753)](https://github.com/httprb/http/commit/9bfcdfb8c97f3e8a319e0a9e3a9e14aef54cef57)
- (a47c436) [Add `base64` to runtime dependency (#759)](https://github.com/httprb/http/commit/a47c436a34a73bbcd36f6630dd604a861b444eb8)
- (4060ccd) [not_found requires two arguments (#761)](https://github.com/httprb/http/commit/4060ccdbe44834bd23e3090a12408aac588a05ad)
- (3b7133c) [fix: close sockets on initialize timeout (#762)](https://github.com/httprb/http/commit/3b7133ca1317ffb897baa2c87eadf3dfd3609fa7)


</details>